Add notes about deprecation to lib-content #443 (#444)
enonic/doc-xp: docs/api/lib-content.adoc

|===
| Name | Type | Attributes| Default| Description
| keys | Array.<string> | | | List of all content keys(path or id) that should be published

> | sourceBranch | string | | | _Deprecated from_ image:xp-7120.svg[XP 7.12.0,opts=inline]. The branch where the content to be published is stored.

Reviewer: ```suggestion
| sourceBranch | string | | | _Not in use since_ image:xp-7120.svg[XP 7.12.0,opts=inline]. The branch where the content to be published is stored.
```

> | sourceBranch | string | | | _Deprecated from_ image:xp-7120.svg[XP 7.12.0,opts=inline]. The branch where the content to be published is stored.

Reviewer 2: ```suggestion
| sourceBranch | string | | | _Not in use from_ image:xp-7120.svg[XP 7.12.0,opts=inline]. The branch where the content to be published is stored.
```

> | targetBranch | string | | | _Deprecated from_ image:xp-7120.svg[XP 7.12.0,opts=inline]. The branch to which the content should be published. Technically, publishing is just a move from one branch to another, and publishing user content from master to draft is therefore also valid usage of this function, which may be practical if user input to a web-page is stored on master

Reviewer: ```suggestion
| targetBranch | string | | | _Not in use since_ image:xp-7120.svg[XP 7.12.0,opts=inline]. The branch to which the content should be published. Technically, publishing is just a move from one branch to another, and publishing user content from master to draft is therefore also valid usage of this function, which may be practical if user input to a web-page is stored on master
```

> | targetBranch | string | | | _Not in use since_ image:xp-7120.svg[XP 7.12.0,opts=inline]. The branch to which the content should be published. Technically, publishing is just a move from one branch to another, and publishing user content from master to draft is therefore also valid usage of this function, which may be practical if user input to a web-page is stored on master

Reviewer 2: ```suggestion
| targetBranch | string | | | _Deprecated from_ image:xp-7120.svg[XP 7.12.0,opts=inline]. The branch to which the content should be published. Technically, publishing is just a move from one branch to another, and publishing user content from master to draft is therefore also valid usage of this function, which may be practical if user input to a web-page is stored on master
```
